Survey/Sources/Supports/Extensions/SwiftUI/Font+.swift

```diff
@@ -23,17 +23,14 @@ extension UIFont {
         var name: String {
             switch self {
             case .regular:
-                return "NeuzeitBook"
+                return "NeuzeitSLTStd-Book"
             case .bold:
-                return "NeuzeitHeavy"
+                return "NeuzeitSLTStd-BookHeavy"
             }
         }
     }
 
-    static func neuzei(style: NeuzeiStyle = .regular, size: CGFloat) -> UIFont {
-        guard let font = UIFont(name: style.name, size: size) else {
-            return UIFont.systemFont(ofSize: size)
-        }
-        return font
+    static func neuzei(style: NeuzeiStyle = .regular, size: CGFloat = 17.0) -> Font {
+        return Font.custom(style.name, size: size)
```

Comment:
I suggest creating style for fonts too.
```
/// Weight: 400 Size: 17 
static let regularBody = Font.neuzei()
```

This way we can private `NeuzeiStyle` and `func neuzei`. We only let dev uses predefined fonts style to conform with Figma design.